### ISSUE TITLE
TS113 Notification for Accredited vendors only

### ISSUE THREAD

**Issue by zetaleona** [vasavah/vsv-fli-pass-uat#40]
•  Notification should be for Accredited vendors only.

**Comment by matthewroman** [comment]
Hannah, to double check, this is referring to the email notification when a vendor is accredited or rejected?

**Comment by HannahCastro** [comment]
Yes, this is for email notification. SAT has decided that only vendors that have been accredited should be notified.

**Comment by matthewroman** [comment]
Noted.  Rejected vendors will NOT receive a notification.

**Comment by HannahCastro** [comment]
Please include that Blacklisted vendors shouldn't get a notice as well. We tried suspending a vendor, and a notification was received.

**Comment by HannahCastro** [comment]
For Rejected vendors, system doesn't notify them. But since there is no Blacklist button yet, we can't test if it also works for these vendors.

**Comment by HannahCastro** [comment]
As of February 20, this issue is not yet resolved.

**Comment by HannahCastro** [comment]
This issue is not yet resolved. A sample vendor received an email regarding suspension.
![image](https://user-images.githubusercontent.com/51902789/90362244-c26d2000-e092-11ea-9706-bbe29305a11d.png)
